src/components/Checkbox/Checkbox.stories.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Checkbox } from "./Checkbox";
+import { withKnobs, text } from "@storybook/addon-knobs";
+
+export default {
+  title: "Checkbox",
+  decorators: [withKnobs],
+  argTypes: {
```

Comment:
Here you're using controls (which applies to all stories from the file) or knobs (used for particular story).

<img width="541" alt="Screenshot 2021-10-30 at 19 05 51" src="https://user-images.githubusercontent.com/20808724/139542405-9caa0c48-5a08-45f5-8805-a8c048dc4f28.png">

I'd recommend to stick with one approach - we may discuss that with other folks 😉